developers/dvl-integration.md

```diff
@@ -0,0 +1,118 @@
+# DVL integration (BETA)
+
+The DVL integration with ArduSub is in a BETA stage and is currently unsupported for normal use. This document is provided to instruct beta testers and developers how to test the BETA stage integration. There is no guarantee that the DVL integration will work for any particular application.
+
+## Supported DVLs
+
+The [Waterlinked A50 DVL](https://waterlinked.com/product/dvl-a50/) is the only supported DVL at the moment in the beta integration.
+
+### A50 DVL configuration
+
+Set it up to use the [Static IP address](https://waterlinked.github.io/docs/dvl/dvl-a50-details/#network-config) 192.168.2.94
```

Comment:
Does this conflict with WL UGPS?

The DVL should be configured with a static ip address of 192.168.2.94. Follow Water Linked's instructions here.

Reply:
YES! that should be .95, my mistake.